setup.py

```diff
@@ -6,6 +6,9 @@
       author="Ewen Cheslack-Postava",
       author_email='ewen@confluent.io',
       platforms=["any"], 
+      entry_points={
+        'console_scripts': ['ducktape=ducktape.__main__:main'],
```

Comment:
Ok, I think I'd just recommend leaving it as is, but trying to keep the amount of code in `__main__.py` from now on.